GOVERNANCE.md

```diff
@@ -166,19 +189,11 @@ Provide a summary of the nominee's contributions. For example:
   organization
 * Other participation in the wider Node.js community
 
-Mention @nodejs/collaborators in the issue to notify other collaborators about
-the nomination.
-
-The nomination passes if no collaborators oppose it after one week. In the case
+The nomination passes if no collaborators oppose it after one week, and if the
```

Comment:
I think we should re-add notifying collaborators mention. This is one of the requirements of each collaborator to follow up, and one of the topics that have a say in.

Reply:
It's still here